### PR TITLE
EXP-635 [extrato] estruturar processo de ci/cd do pacote balance

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
Adiciona estrutura de CI para realizar o build no novo projeto Balance

## Checklist
- [x] Corrige o eslint no balance
- [x] Adiciona lint para disparar no ci 
- [x] Adiciona build para o balance em Sandbox na AWS
- [x] Adiciona build para o balance em Live na AWS

## Screenshots
image

### CODE REVIEW

**Review by Hrfreire — APPROVED** [review]

File: packages/balance/src/index.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ import 'normalize.css'
 import App from './App'
 
 const lifecycles = singleSpaReact({
+  // eslint-disable-next-line no-unused-vars
+  errorBoundary (err, info, props) {
```
[the remark this answers is not in the document]
Comment (Hrfreire) [reply packages/balance/src/index.js:10]: Essa função tinha sido removida pelo PR de tratamento de erros. Não é mais necessário declarar esse errorBoundary. Acho que ela acabou sendo declarada novamente.

File: packages/balance/src/pages/Root.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ const Root = ({
         session_id: sessionId,
       })
     }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```
[the remark this answers is not in the document]
Comment (diegodsgarcia) [reply packages/balance/src/pages/Root.js]: Confesso que não testei 100%, porém pelos poucos testes que fiz aqui na dash da mundi não teve nenhum efeito negativo com as props, então adicionei elas :)

**Review by LucasRichter — APPROVED** [review]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbf78d1a99fe9e3f27ffc06515f73d790d33c7b0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbf78d1a99fe9e3f27ffc06515f73d790d33c7b0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbf78d1a99fe9e3f27ffc06515f73d790d33c7b0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbf78d1a99fe9e3f27ffc06515f73d790d33c7b0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by Hrfreire — APPROVED** [review]

**Pull request merged**